views/shelf.erb

```diff
@@ -6,14 +6,33 @@
 
 <% elsif %>
 
+<div class="renew-all-message">
+  <div class="renew-all-message-content">
+    <p><strong>15 items were successfully renewed.</strong></p>
```

Comment:
@niquerio could you please replace `15` and `12` with back-end data? Thank you!

Reply:
@niquerio and make this message conditional on the renew all message being available from the back-end?